### ISSUE TITLE
Insufficient color contrast on harmful language statement

### ISSUE THREAD

**Issue by nrsa222** [uklibraries/findingaid#26]
.fa-toc > .alert-harmful.alert > p:nth-child(2) > a[href$="mailto:SCRC@uky.edu"]

Element has insufficient color contrast of 2.68 (foreground color: #337ab7, background color: #b1c9e8, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1.

- [ ] Update the link in this block to to display as an underline link
- [ ] Move block to global footer, so that it displays on every page but doesn't take up valuable real estate that could be use for the "cart" or other content.
